common/beerocks/btl/btl.cpp

```diff
@@ -41,16 +41,23 @@ void transport_socket_thread::set_select_timeout(unsigned msec) { poll_timeout_m
 bool transport_socket_thread::send_cmdu_to_bus(ieee1905_1::CmduMessageTx &cmdu_tx,
                                                const std::string &dst_mac,
                                                const std::string &src_mac)
+{
+    return send_cmdu_to_bus(cmdu_tx, 0, dst_mac, src_mac);
+}
+
+bool transport_socket_thread::send_cmdu_to_bus(ieee1905_1::CmduMessageTx &cmdu_tx,
+                                               uint32_t iface_index, const std::string &dst_mac,
```

Comment:
Why not add the `iface_index` as the last parameter with a default value of 0?

Reply:
Because there is another method in the same class with the same name and parameters but with an extra parameter `uint16_t length` at the end.
Before my changes there were these two methods in `transport_socket_thread`:
```c
public:
bool send_cmdu_to_bus(ieee1905_1::CmduMessage &cmdu, const std::string &dst_mac, const std::string &src_mac);
protected:
bool send_cmdu_to_bus(ieee1905_1::CmduMessage &cmdu, const std::string &dst_mac, const std::string &src_mac, uint16_t length);
```

Reply:
As requested in this [comment](https://github.com/prplfoundation/prplMesh/pull/1420#discussion_r441604127), iface_index has been replaced by iface_name and added as the last parameter with a default value of ""